pdata/pcommon/common.go

```diff
@@ -50,7 +50,7 @@ func (avt ValueType) String() string {
 	case ValueTypeEmpty:
 		return "EMPTY"
```

Comment:
A good question is if we should use uppercase or not here? I feel this can be `Empty` etc. Consistent with other enums. Will file an issue.

Reply:
I'm actually writing that issue currently :)

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/6247